### ISSUE TITLE
Pre-trained weights

### ISSUE THREAD

**Issue by zimonitrome** [maudzung/TTNet-Real-time-Analysis-System-for-Table-Tennis-Pytorch#10]
It would be useful if pre-trained weights could be provided.

Are there any plans to making them available?

**Comment by koren88i** [comment]
yes please I do not have enough cpu or storage to prepare dataset and make training..

**Comment by elisim** [comment]
pre-trained model would be very useful

**Comment by koren88i** [comment]
https://filebin.net/nvfumr1yo943q3j8
1. Trained TTNet with default params except batch size of 4 isntead of 8. Best Model out of 30 epochs.
2. Logs for my train and test.

**Comment by rudyn2** [comment]
Hi, @koren88i could you re-upload the pretrained weights please, i don't have the resources to train ttnet.

**Comment by JIANG-CX** [comment]
@koren88i Could you please provide the pretrained weights again? The link before was expired. Thanks a lot.

**Comment by liujunlj** [comment]
@koren88i Could please kindly help to provide the pre-trained model? We cannot follow the above link to get the file. Thanks a lot!

**Comment by koren88i** [comment]
my trained model did not work... 0.6 ball bounce all the time. If I succeed I will upload again.
Sorry for the delay.

**Comment by liujunlj** [comment]
got it. thanks!

**Comment by egehancosgun** [comment]
> my trained model did not work... 0.6 ball bounce all the time. If I succeed I will upload again.
> Sorry for the delay.

Did you succeed now? I also have similar problems. I implemented the same training strategy of the repo but I could not get accurate predictions.

**Comment by alamayreh** [comment]
Hi all, 

Can I get pertained models? 

Thanks

**Comment by abedall** [comment]
hi guys anyone get the petrained model?? please

**Comment by abedall** [comment]
> my trained model did not work... 0.6 ball bounce all the time. If I succeed I will upload again.
> Sorry for the delay.

hi my friend ..can you help me how did you trained the model please ..the same strategy they mentioned in their project?

**Comment by manilpit** [comment]
Could anyone provide pretrained weights please?

**Comment by ChristiaensBert** [comment]
Do you have the pretrained weights?

**Comment by ivanshin** [comment]
Hi, can anyone share weights for me ?

**Comment by Squirrel8475** [comment]
Hello, I ran the training and testing with the settings available in the files and could not reproduce the results either. The segmentation of the players is OK, but not the ball detection. Most of time there is no detection, and if a detection occurs, the position is always around the middle of the image far away from ground truth. The problem occurs already at the Global stage (1st training).

**Comment by ratnakarkambham** [comment]
> Hello, I ran the training and testing with the settings available in the files and could not reproduce the results either. The segmentation of the players is OK, but not the ball detection. Most of time there is no detection, and if a detection occurs, the position is always around the middle of the image far away from ground truth. The problem occurs already at the Global stage (1st training).

i am facing the same issue?

**Comment by YannickGibson** [comment]
> my trained model did not work... 0.6 ball bounce all the time. If I succeed I will upload again. Sorry for the delay.

@koren88i Hi, have you succeeded?
